backend/sirius-components-emf/src/main/java/org/eclipse/sirius/components/emf/view/OperationInterpreter.java

```diff
@@ -108,37 +113,37 @@ private Optional<VariableManager> executeOperation(Operation operation, Variable
         ViewSwitch<Optional<VariableManager>> dispatcher = new ViewSwitch<>() {
             @Override
             public Optional<VariableManager> caseChangeContext(ChangeContext op) {
-                return ToolInterpreter.this.executeChangeContext(variableManager, op);
+                return OperationInterpreter.this.executeChangeContext(variableManager, op);
```

Comment:
Can you extract this switch in a dedicated class please.

Reply:
Extracted into two switches: one for common operation (no dependencies to a specific representation) and a second for diagram specific operations: createView and deleteView.